modules/influxdb/src/main/java/org/testcontainers/containers/InfluxDBContainer.java

```diff
@@ -1,163 +1,10 @@
-        return Collections.singleton(getMappedPort(INFLUXDB_PORT));
-    }
-
-    /**
-     * Set env variable `INFLUXDB_HTTP_AUTH_ENABLED`.
-     *
-     * @param authEnabled Enables authentication.
-     * @return a reference to this container instance
-     */
-    public SELF withAuthEnabled(final boolean authEnabled) {
-        this.authEnabled = authEnabled;
-        return self();
-    }
-
-    /**
-     * Set env variable `INFLUXDB_ADMIN_USER`.
-     *
-     * @param admin The name of the admin user to be created. If this is unset, no admin user is created.
-     * @return a reference to this container instance
-     */
-    public SELF withAdmin(final String admin) {
-        this.admin = admin;
-        return self();
-    }
-
-    /**
-     * Set env variable `INFLUXDB_ADMIN_PASSWORD`.
-     *
-     * @param adminPassword TThe password for the admin user configured with `INFLUXDB_ADMIN_USER`. If this is unset, a
-     *                      random password is generated and printed to standard out.
-     * @return a reference to this container instance
-     */
-    public SELF withAdminPassword(final String adminPassword) {
-        this.adminPassword = adminPassword;
-        return self();
-    }
-
-    /**
-     * Set env variable `INFLUXDB_DB`.
-     *
-     * @param database Automatically initializes a database with the name of this environment variable.
-     * @return a reference to this container instance
-     */
-    public SELF withDatabase(final String database) {
-        this.database = database;
-        return self();
-    }
-
-    /**
-     * Set env variable `INFLUXDB_USER`.
-     *
-     * @param username The name of a user to be created with no privileges. If `INFLUXDB_DB` is set, this user will
-     *                 be granted read and write permissions for that database.
-     * @return a reference to this container instance
-     */
-    public SELF withUsername(final String username) {
-        this.username = username;
-        return self();
-    }
-
-    /**
-     * Set env variable `INFLUXDB_USER_PASSWORD`.
-     *
-     * @param password The password for the user configured with `INFLUXDB_USER`. If this is unset, a random password
-     *                 is generated and printed to standard out.
-     * @return a reference to this container instance
-     */
-    public SELF withPassword(final String password) {
-        this.password = password;
-        return self();
-    }
-
-
-    /**
-     * @return a url to influxDb
-     */
-    public String getUrl() {
-        return "http://" + getHost() + ":" + getLivenessCheckPort();
-    }
-
-    /**
-     * @return a influxDb client
-     */
-    public InfluxDB getNewInfluxDB() {
-        InfluxDB influxDB = InfluxDBFactory.connect(getUrl(), username, password);
-        influxDB.setDatabase(database);
-        return influxDB;
+@Deprecated
+public class InfluxDBContainer<SELF extends InfluxDBContainer<SELF>> extends InfluxDBContainerV1<SELF> {
```

Comment:
It might be helpful to include a Javadoc with a `@deprecated` attribute that instructs people to use either `InfluxDBContainerV1` or `InfluxDBContainerV2` instead.